src/app/modules/shared/components/details-fields/details-fields.component.html

```diff
@@ -34,108 +43,112 @@
     </div>
   </div>
 
-
-  <div class="form-group row">
-    <label class="col-12 col-sm-2 col-form-label">Activity:</label>
-    <div class="col-12 col-sm-10">
-      <select
-        [class.is-invalid]="activity_id.invalid"
-        required
-        id="activity_id"
-        class="custom-select"
-        formControlName="activity_id"
-      >
-        <option value="" selected="selected"></option>
-        <option *ngFor="let activity of activities" value="{{ activity.id }}">{{ activity.name }}</option>
-      </select>
+  <fieldset [disabled]="!(project_id.value && project_name.value)">
+    <div class="form-group row">
+      <label class="col-12 col-sm-2 col-form-label">Activity:</label>
+      <div class="col-12 col-sm-10">
+        <select
+          [class.is-invalid]="activity_id.invalid"
```

Comment:
I know this is not part of your ticket. But let's improve the activity field feedback by changing this line to `[class.is-invalid]="activity_id.invalid && activity_id.touched"`